src/components/LoginCard/LoginCard.js

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import GoogleLogin from 'react-google-login';
-import {GOOGLE_APP_ID, EGO_API} from '../../common/globals';
```

Comment:
It looks like maybe prettier didn't pick up the prettier config file in the root of the project, or maybe we are missing some configuration setting?